packages/slice-machine/pages/_app.tsx

```diff
@@ -110,7 +111,7 @@ function MyApp({ Component, pageProps }: AppContext & AppInitialProps) {
               <LoadingPage />
             ) : (
               <Provider store={smStore.store}>
-                <ConnectedRouter>
+                <ConnectedRouter Router={Router}>
```

Comment:
why did we needed that ?

Reply:
it was a suggestion from Marc he might be able to explain better. But it's basically to synch the next/router to the connected router from `connected-next-router` that we use to store the url location in redux

Reply:
ok :)